docs/setup-cluster/k8s/controller-reviews.rst

```diff
@@ -0,0 +1,168 @@
+.. _controller-reviews:
+
+###############################
+ Gateway API Controller Survey
+###############################
+
+https://gateway-api.sigs.k8s.io/implementations/#haproxy-kubernetes-ingress-controller
+
+*********
```

Comment:
did we test with all of these?

Reply:
no so we probably want to clarify that. I did test with Envoy gateway and halfway with Epic, maybe we categorize those differently?